apps/system/utils/kdbg_heapinfo.c

```diff
@@ -110,6 +136,26 @@ int kdbg_heapinfo(int argc, char **args)
 	printf("\n** NOTED **\n");
 	printf("* Idle Task's stack is not allocated in heap region\n");
 	printf("* And another stack size is allocated stack size + alloc node size(task:32/pthread:20)\n\n");
+
+#ifdef CONFIG_HEAPINFO_GROUP
+	if (show_group == true) {
+		int group_idx;
+		printf("****************************************************************\n");
+		printf("Heap Allocation Information per User defined Group\n");
+		printf("****************************************************************\n");
+		printf(" PEAK | HEAP_WHEN_PEAK | STACK_WHEN_PEAK | THREADS_IN_GROUP \n");
```

Comment:
HEAP WHEN PEAK -> Heap on peak.
STACK WHEN PEAK -> STACK,
on current implementation, stack is not the usage of stack. it's summation of stack size.